alexxsanya/epic-mail-api
# 164979422 ES6 & PEP8 Standard Refactor

**What does this PR do ?**
- Refactor JS to ES6 stands
- Refactor tests to PEP8 standards
- Fix failing tests.

**How to test this PR manually?**
- kindly check changed files
- for the fixed failing tests click [here](https://travis-ci.org/alexxsanya/EPIC-MAIL-API)
